ipn/ipnlocal/local.go

```diff
@@ -213,8 +217,7 @@ type LocalBackend struct {
 	state          ipn.State
 	capFileSharing bool // whether netMap contains the file sharing capability
 	capTailnetLock bool // whether netMap contains the tailnet lock capability
-	// hostinfo is mutated in-place while mu is held.
-	hostinfo *tailcfg.Hostinfo
+	hostinfo       tailcfg.HostinfoView
```

Comment:
comment about whether this includes the blended-in fields, and which?

Reply:
done